### PR TITLE
Chef13 deprecations

### PR DESCRIPTION
**Pull request by chrisminton**
- Changes to remove the deprecation warnings listed running Chef 13
- Test framework overhaul - use Inspec rather than Serverspec, update, add to and streamline unit tests
- Updates to READMEs, copyrights & notices
- Rubocop fixes
- Add a fix to start `websphere_cluster_member`
- add timeout option for Deployment Manager `startManager`

### CODE REVIEW

File: libraries/ibm_cert.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #
 # Cookbook Name:: websphere
+# Resource:: ibm_cert
 #
-# Copyright (C) 2015 J Sainsburys
+# Copyright (C) 2015-2018 J Sainsburys
```
Comment (tad78) [inline libraries/ibm_cert.rb]: 2019?

File: libraries/websphere_app.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #
 # Cookbook Name:: websphere
-# Resource:: websphere-server
+# Resource:: websphere_app
 #
-# Copyright (C) 2015 J Sainsburys
+# Copyright (C) 2015-2018 J Sainsburys
```
Comment (tad78) [inline libraries/websphere_app.rb]: 2019 - see I'm definitely adding value here... ;)

File: libraries/websphere_base.rb [hunk 3]
```diff
@@ -132,18 +132,18 @@ def sync_node_wsadmin(nde_name = 'all', bin_directory = '/opt/IBM/WebSphere/AppS
...
       # TODO: add check if node or server are already started first.
-      def start_manager(p_name, profile_env, profile_bin = bin_dir)
+      def start_manager(p_name, profile_env, profile_bin = new_resource.bin_dir)
         # start_profile(profile_name, bin_dir, "./startManager.sh -profileName #{profile_name}", [0, 255])
         execute "start dmgr profile: #{p_name}" do
           cwd profile_bin
           environment profile_env
-          user run_user
+          user new_resource.run_user
           command "./startManager.sh -profileName #{p_name}"
```
Comment (tad78) [inline libraries/websphere_base.rb]: should we add the timeout option?

File: libraries/websphere_cluster.rb [hunk 4]
```diff
@@ -1,8 +1,8 @@
 #
 # Cookbook Name:: websphere
-# Resource:: websphere-server
+# Resource:: websphere_cluster
 #
-# Copyright (C) 2015 J Sainsburys
+# Copyright (C) 2015-2018 J Sainsburys
```
Comment (tad78) [inline libraries/websphere_cluster.rb]: 19

File: libraries/websphere_base.rb [hunk 5]
```diff
@@ -43,20 +43,20 @@ class WebsphereBase < Chef::Resource
...
+          user new_resource.run_user
+          group new_resource.run_group
           command command
-          sensitive sensitive_exec
+          sensitive new_resource.sensitive_exec
           action :run
         end
       end
 
       # returns a hash of enpoint => port for a server
```
Comment (tad78) [inline libraries/websphere_base.rb]: endpoint?

File: libraries/websphere_base.rb (hunk 3 above)
Comment (chrisminton) [reply libraries/websphere_base.rb]: Done @tad78

**Pull request merged**